cynthion/python/src/gateware/analyzer/analyzer.py

```diff
@@ -546,8 +561,13 @@ def instantiate_dut(self):
         m = Module()
         m.submodules.translator = self.translator = UTMITranslator(ulpi=self.ulpi, handle_clocking=False)
         m.submodules.analyzer   = self.analyzer   = USBAnalyzer(utmi_interface=self.translator, mem_depth=128)
-        m.submodules.s16to8 = s16to8 = DomainRenamer("usb")(Stream16to8())
-        m.d.comb += s16to8.input.stream_eq(self.analyzer.stream)
+        reset_on_start = ResetInserter(self.analyzer.discarding)
+        m.submodules.clk_conv = clk_conv = StreamSyncUsbConverter()
```

Comment:
I would add `reset_on_start` here too: the output of this block is registered, so it can buffer single words while `analyzer.discarding` is active.